.github/workflows/publish-release.yaml

```diff
@@ -68,9 +68,9 @@ jobs:
         uses: peter-evans/create-pull-request@v6
         with:
           # Note: `publish-release.yaml` checks explicitly for this commit message
-          commit-message: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
-          title: 'chore: release tutorialkit CLI, version: ${{ steps.resolve-release-version.outputs.version }}'
-          body: 'Bump tutorialkit CLI to version ${{ steps.resolve-release-version.outputs.version }}'
+          commit-message: 'chore: release TutorialKit CLI v${{ steps.resolve-release-version.outputs.version }}'
```

Comment:
Commit message is used here:

https://github.com/stackblitz/tutorialkit/blob/8b8b1caea8793748d9946e163a184a3ecb958358/.github/workflows/publish-release.yaml#L84

Reply:
What do you mean? I am confused 😅

Reply:
We'll need to change that line 84 to match the new changed commit message as well:

```diff
- if: ${{ contains(github.event.head_commit.message, 'release tutorialkit CLI') }
+ if: ${{ contains(github.event.head_commit.message, 'release TutorialKit CLI') }
```

Reply:
Ah got ya! I didn't understand that from your message that I had to change something, sorry. I ll update that as well. That's for the pointer.